composer.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
please remove this file, it shouldn't be included in this PR

Reply:
oops! it shouldn't have been committed, sorry...